README.md

```diff
@@ -2,3 +2,91 @@
 
 Automation tools for deploying blockchain networks
 
+## Prerequisites
+Prior to using this script, it's required that docker, sbt, and openjdk are installed on the host.  
+In order to install Docker, please execute the following commands:
```

Comment:
Let's make it clear these instructions are for Ubuntu and some commands might need to be modified for other operating systems. Better yet, just list the packages the user has to install and let them figure out the steps on their own.